src/server/src/routes/job.ts

```diff
@@ -121,7 +121,7 @@ router.get(
             numOfItems,
         } = req.query as {
             title: string;
-            type: string;
+            type: string[];
```

Comment:
Does "type" really come as a string[]? I thought it would have to be a string since it's deconstructed from the query parameters which is just a string.

Reply:
That's what I thought but when I use string instead of string[], it gives me this Error: Material-UI: The `value` prop must be an array when using the `Select` component with `multiple`.